### ISSUE TITLE
Question - do you have a standalone windows binary?

### ISSUE THREAD

**Issue by gjanders** [unxsist/jet-pilot#25]
I wanted to test jet pilot, however, the setup shows "Unknown publisher" and is blocked by Microsoft Defender

Do you have an option to run the jet pilot app without installing it locally?

Other utilities appear to be able to install in "user" mode and were not blocked...this might be a policy coming from my work laptop

**Comment by unxsist** [comment]
Currently I don't have a standalone build for Windows, but even with the standalone build you might face similar issues. The reason MS Defender is blocking it, is because the Windows releases are currently not signed. Currently only the MacOS releases of JET Pilot are signed and notarized as that's the OS I work with myself.

Sadly both Apple and MS made it quite expensive to have code-signing (even for open-source and free software), JET Pilot is still a relatively small project and I paid the Apple Developer License myself to improve usability on MacOS. I might consider doing the same for Windows if the user-base grows over time.

**Comment by gjanders** [comment]
Thanks, I was unaware there is a cost involved

**Comment by unxsist** [comment]
You could however try building JET Pilot from source? It should be a relatively straight forward process. Happy to provide build instructions for Windows if you run into issues.

**Comment by gjanders** [comment]
While it took a bit of effort, compiling from source and then running the exe offered to install it locally
It appears to be (mostly) working and I'll open a new issue if I need help

Thankyou!

**Comment by unxsist** [comment]
Great to hear! Feel free to report any issues you're experiencing, even though it's a local build!
